glean-core/android-native/proguard-rules-consumer.pro

```diff
@@ -0,0 +1,7 @@
+# ProGuard rules for consumers of this library.
+
+# JNA specific rules
+# See https://github.com/java-native-access/jna/blob/master/www/FrequentlyAskedQuestions.md#jna-on-android
+-dontwarn java.awt.*
+-keep class com.sun.jna.* { *; }
+-keepclassmembers class * extends com.sun.jna.* { public *; }
```

Comment:
Should we retain the same things we retain for the [`android` package](https://searchfox.org/glean/source/glean-core/android/proguard-rules-consumer.pro#15)?

Reply:
I'll check, you might be right.

Reply:
We don't need to, the native package doesn't export any namespaces.